PubHub/PubHub.API/Domain/Entities/BookGenre.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace PubHub.API.Domain.Entities
+{
+    public class BookGenre
+    {
+        public int BookId { get; set; }
+        public int GenreId { get; set; }
+
+        #region Navs
+        public Book? Book { get; set; }
+        public Genre? Genre { get; set; }
```

Comment:
Same as [this comment](https://github.com/Mike-Mortensen-Portfolio/PubHub_H6_Final/pull/65/files#r1542453037).